### PR TITLE
UHF-9088: Translations

### PR DESCRIPTION
**Pull request by khalima**
# [UHF-9088](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9088)

## What was done

* Updated translations based on UI translation updates in Strategia -instance and content designer's suggestions

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-9088 drupal/hdbt:dev-UHF-9088 drupal/helfi_tpr:dev-UHF-9088 drupal/helfi_api_base:dev-UHF-9088 drupal/hdbt_admin:dev-UHF-9088 -W`
* Run `make drush-updb drush-cr`
* Run `make shell`
    * In shell, run: `drush locale:check; drush locale:update; drush helfi:locale-import helfi_platform_config drush cr`

## How to test

* [x] Check that the configuration translations get updated when you run `make drush-cex`
* [x] Test for changed translations in "spot check" fashion.. I.e. Check for a changed translation in helfi_platform_config/helfi_tpr/hdbt `fi.po` file and check from source code where it's in use and check that the translation has been changed.
* [x] Check that code follows our standards

## Other PRs

* https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/170
* https://github.com/City-of-Helsinki/drupal-hdbt/pull/918
* https://github.com/City-of-Helsinki/drupal-module-helfi-api-base/pull/152
* https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/275

[UHF-9088]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-9088?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/hdbt_admin_tools/translations/fi.po [hunk 1]
```diff
@@ -26,25 +26,15 @@ msgctxt "HDBT Admin editorial"
 msgid "Title"
 msgstr "Sivun otsikko"
 
-msgctxt "HDBT Admin editorial"
```
Comment (khalima) [inline modules/hdbt_admin_tools/translations/fi.po:29]: This translation was a duplicate.

File: modules/helfi_eu_cookie_compliance/translations/fi.po [hunk 2]
```diff
@@ -1,1198 +1,6 @@
 msgid ""
 msgstr ""
 
-msgid ""
```
Comment (khalima) [inline modules/helfi_eu_cookie_compliance/translations/fi.po:4]: The cookie compliance texts do not live in .po files, so these can be removed.

File: modules/helfi_eu_cookie_compliance/translations/sv.po [hunk 3]
```diff
@@ -1,1213 +1,6 @@
 msgid ""
 msgstr ""
 
-msgid ""
```
Comment (khalima) [inline modules/helfi_eu_cookie_compliance/translations/sv.po:4]: The cookie compliance texts do not live in .po files, so these can be removed.

**Review by teroelonen — APPROVED** [review]
Amazing work with this one - you are simply awesome <3 🦖

**Pull request merged**